src/LogicalGraph.ts

```diff
@@ -131,13 +131,27 @@ export class LogicalGraph {
 
     static toOJSJsonString(graph : LogicalGraph, forTranslation : boolean) : string {
         let result: string = "";
-
         const json: any = LogicalGraph.toOJSJson(graph, forTranslation);
 
         // NOTE: manually build the JSON so that we can enforce ordering of attributes (modelData first)
         result += "{\n";
         result += '"modelData": ' + JSON.stringify(json.modelData, null, 4) + ",\n";
-        result += '"graphConfigurations": ' + JSON.stringify(json.graphConfigurations, null, 4) + ",\n";
+
+        // if we are sending this graph for translation, then only provide the "active" graph configuration, or an empty array if none exist
```

Comment:
I think that some users might intend to use the default values specified in the base graph. If, for example, the graph was created for a single purpose, without any requirement to be modified for multiple runs.

In that case a user would not need any graph configurations, so none would be active.

We could either REQUIRE a graph configuration, and it would simple overwrite zero parameter values, or allow an empty graph configurations array. I think the later should be supported.

Reply:
I see what you mean, and you've convinced of the approach. Thank you for explaining things through.